lib/ClangImporter/ImportDecl.cpp

```diff
@@ -8066,7 +8072,7 @@ ClangImporter::Implementation::importDeclContextOf(
   switch (context.getKind()) {
   case EffectiveClangContext::DeclContext: {
     auto dc = context.getAsDeclContext();
-    if (dc->isTranslationUnit()) {
+    if (dc->isFileContext()) {
```

Comment:
Why is this change needed?

Reply:
This is needed to support namespaces. Decls inside a namespace are treated as toplevel. This is not used for purposes of lookup (you have to alias the name into the toplevel scope in order for it to be findable).